autoload/all-all/woofhud.lmp

```diff
@@ -4,6 +4,7 @@ monsec topleft
 sttime topleft
 coord topright
 fps topright
+cmd bottomright
```

Comment:
Currently, multi-line widgets don't work properly when using absolute screen coordinates.

Reply:
No, unfortunately not.

Reply:
Should we consider this out of scope for this PR?

Reply:
I agree it should be fixed, but this is orthogonal to the work here.